gui/src/app/prototypes/AnalysisPyPrototype/AnalysisPyPrototype.tsx

```diff
@@ -0,0 +1,93 @@
+import { Splitter } from "@fi-sci/splitter"
+import { FunctionComponent, useEffect, useState } from "react"
+import AnalysisPyFileEditor from "./AnalysisPyFileEditor"
+
+type AnalysisPyPrototypeProps = {
+    width: number
+    height: number
+}
+
+const exampleScript = `import sys
```

Comment:
Just to note, the `stanio` package will probably also be helpful here in terms of preparing a useful output object from the draws. You can see what I'm doing in the normal python tinystan here:

https://github.com/WardBrian/tinystan/blob/main/clients/python/tinystan/output.py

Reply:
I'm wondering whether we should use a pandas dataframe (long form, all chains in one). We want something that will stand the test of time so that old SP projects won't stop working.

Reply:
If you want compatibility with existing packages, your best bet will either be something like the tinystan object (which is basically just a dict with ndarray entries), or for maximum interoperability something like ArViz's inferencedata object

Reply:
I think simpler is better. And we should also try to make it as consistent as possible between R and Python.

Reply:
R is a slightly different story, but it has a similar sort of lock-in by a package called `posterior`. If whatever we provide is not at least easily convertible to something posterior can understand, it won't be used

Reply:
One approach could be to provide the data to the script in as raw a format as possible. Then have a standard first few of lines that load it into posterior, or arviz, or pandas, etc. Those conversion lines would be part of the analysis.py/analysis.r.

Reply:
That seems fine to me